src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -68,8 +71,19 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-        return new CommandResult(String.format(MESSAGE_TAG_ADD_SUCCESS,
-                editedPerson.getName(), editedPerson.getTags()));
+
+        Set<Tag> oldTags = personToEdit.getTags();
+        Predicate<Tag> tagInOldTags = testTag -> oldTags.contains(testTag);
+
+
+        Set<Tag> overlapTags = editPersonDescriptor.getTags().orElse(new HashSet<>()).stream()
+                .filter(tagInOldTags).collect(Collectors.toSet());
+
+        String successMessage = String.format(MESSAGE_TAG_ADD_SUCCESS, editedPerson.getName(), editedPerson.getTags());
```

Comment:
Just a thought will it look weird? Like the message will say that this tag has been added, but also say that it already exist?

Reply:
I think maybe a better way is for those that don't overlap, have message tag add success, while those which do, have message_tag_add_exists. 
Eg. tag t/hi t/lol. If the person already has t/hi, then the mesasage should be lol successfully added. hi is already added. What do you think?